extensions/EXT/EGL_EXT_device_query_vendor.txt

```diff
@@ -0,0 +1,77 @@
+Name
+
+    EXT_device_query_vendor
+
+Name Strings
+
+    EGL_EXT_device_query_vendor
+
+Contributors
+
+    Eric Engestrom (eric 'at' engestrom.ch)
+
+Contact
+
+    Eric Engestrom (eric 'at' engestrom.ch)
+
+Status
+
+    Draft
+
+Version
+
+    Version 1, Jun 06, 2020
+
+Number
+
+    EGL Extension #140
+
+Extension Type
+
+    EGL device extension
+
+Dependencies
+
+    This extension is written against the language of EGL 1.5 as
+    modified by EGL_EXT_device_query.
+
+    EGL_EXT_device_query is required.
+
+Overview
+
+    This extension adds a simple way to query the PCI vendor ID and device ID.
+
+IP Status
+
+    No known claims.
+
+New Types
+
+    None.
+
+New Procedures and Functions
+
+    None.
+
+New Tokens
+
+    None.
+
+Changes to section 3.2 (Devices)
+
+    Replace the paragraph immediately following the prototype for
+    eglQueryDeviceAttribEXT:
+
+    <attribute> may be either EGL_QUERY_VENDOR_ID_MESA or EGL_QUERY_DEVICE_ID_MESA.
```

Comment:
Oh, don't you mean `EGL_VENDOR or EGL_DEVICE_EXT` here?

Reply:
New attributes would make more sense. `EGL_VENDOR` is currently used to query a string, and `EGL_DEVICE_EXT` always corresponds to an EGLDeviceEXT handle. The PCI vendor and device ID's are both 16-bit integers.

Reply:
I made new attributes originally, but then I decided to use the existing ones as they are unambiguous in the `eglQueryDeviceAttribEXT()` context. I did forget to fix it here, good catch :upside_down_face:

I'm happy to go back to new attributes if that's the consensus :+1:

Reply:
I've fixed it with the re-used attributes for now, but if a second person agrees that we should be adding new enums instead I'll change that everywhere

Reply:
I'll make new enums for `EGL_QUERY_PCI_{VENDOR,DEVICE}_EXT` since I need to make new ones for `EGL_QUERY_PCI_{DOMAIN,BUS,SLOT,FUNCTION}_EXT` anyway